### DISCUSSION TITLE
Meta releasing a new llama model soon

### DISCUSSION THREAD

**Discussion by ianscrivener** [ggml-org/llama.cpp#2226] [General]
Some reports around that Meta is releasing a new & improved llama model soon. It will be "available for commercial use".

"The competitive landscape of AI is going to completely change in the coming months, in the coming weeks maybe, when there will be open source platforms that are actually as good as the ones that are not [opensource],” **vice-president and chief AI scientist at Meta**, **Yann LeCun**, said at a conference in Aix-en-Provence last Saturday.
	
- [The Information](https://www.theinformation.com/articles/meta-wants-companies-to-make-money-off-its-open-source-ai-in-challenge-to-google)
- [Financial Times](https://www.ft.com/content/01fd640e-0c6b-4542-b82b-20afb203f271)

**Comment by tfius** [comment]
llama2 is out. Is support planned ?

**Reply by ianscrivener** [reply]
I'd bet GG and the inner-circle C/C++ devs have been working on it for a couple of weeks 😮

**Reply by Green-Sky** [reply]
afraid not

**Reply by ianscrivener** [reply]
You _would_ say that. "The first rule of Fight Club is: you do not talk about Fight Club." 😉

**Comment by maddes8cht** [comment]
@TheBloke already does provide ggml-versions of the 7b and 13b llama-2 variants.
https://huggingface.co/TheBloke/Llama-2-7B-GGML
https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGML
https://huggingface.co/TheBloke/Llama-2-13B-GGML
https://huggingface.co/TheBloke/Llama-2-13B-chat-GGML

There is also a new 70b-variant, close to what 65b was, but nothing near  the former 33b. We will see if 70b requires further work until these work in llama.cpp or if it just takes more time to convert them...

The context size of these new models is 4k from start, without any tricks.

**Comment by ianscrivener** [comment]
Yep, llama-2-7b-chat.ggmlv3.q4_0.bin works for me!

**Comment by TheNotary** [comment]
Here's a detailed writeup from TheBloke's community discord if it helps clarify anything.  
<img width="1160" alt="image" src="https://github.com/ggerganov/llama.cpp/assets/799247/c4c5739b-dd1b-4eb2-b29b-f9f7f5be26ab">
